### PR TITLE
Show the top 9 most recent reconstruction in explore page

### PR DESCRIPTION
**Pull request by KGhising**
- show the top 9 most recent reconstruction in explore page
- card style change on ReconstructionsCard

### CODE REVIEW

File: src/Components/ReconstructionCard.jsx [hunk 1]
```diff
@@ -26,17 +26,20 @@ class ReconstructionCard extends React.Component {
...
+                  {this.props.reconstruction.name}
+                </span>
               </h5>
-              {this.props.showCreator && (
-                <p className='creator-user-name'>
-                  Created by{this.props.reconstruction.createdByUser.username}
-                </p>
-              )}
+              <p>
+                Created by{' '}
```
Comment (silwalanish) [inline src/Components/ReconstructionCard.jsx:35]: ???

Comment (silwalanish) [reply src/Components/ReconstructionCard.jsx:35]: Where is showCreator logic????

File: src/Pages/Explore.jsx [hunk 2]
```diff
@@ -1,13 +1,78 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true,
+      reconstruction: [],
+    };
+    this.reconstructionCardFetchHandler = this.reconstructionCardFetchHandler.bind(
+      this
+    );
```
Comment (silwalanish) [inline src/Pages/Explore.jsx:22]: Not needed.

File: src/Pages/Explore.jsx [hunk 3]
```diff
@@ -1,13 +1,78 @@
...
+  }
+
+  componentDidMount() {
+    this.reconstructionCardFetchHandler();
+  }
+
+  async reconstructionCardFetchHandler() {
```
Comment (silwalanish) [inline src/Pages/Explore.jsx:29]: rename to fetchReconstructions

**Review by silwalanish — CHANGES_REQUESTED** [review]

File: src/Components/ReconstructionCard.jsx (hunk 1 above)
Comment (KGhising) [reply src/Components/ReconstructionCard.jsx:35]: I think showCreator is not necessary. Does Construction without username exist?

File: src/Pages/Explore.jsx (hunk 2 above)
Comment (KGhising) [reply src/Pages/Explore.jsx:22]: Okay

File: src/Pages/Explore.jsx (hunk 3 above)
Comment (KGhising) [reply src/Pages/Explore.jsx:29]: Okay

**Review by SG-coder — APPROVED** [review]

**Review by SG-coder — APPROVED** [review]

File: src/Pages/Explore.jsx [hunk 4]
```diff
@@ -25,21 +90,23 @@ class Explore extends React.Component {
...
       </React.Fragment>
     );
   }
 }
 
+Explore.propTypes = {
+  reconstruction: PropTypes.object,
+  loading: PropTypes.bool,
+};
+
```
Comment (silwalanish) [inline src/Pages/Explore.jsx:111]: ???

File: src/Components/ReconstructionCard.jsx [hunk 5]
```diff
@@ -26,17 +26,24 @@ class ReconstructionCard extends React.Component {
         <div className='reconstruction-card-details'>
           <div className='card-details'>
             <div>
-              <h5 className='reconstruction-card-title'>
-                Project{this.props.reconstruction.name}
+              <h5
+                className={this.props.showCreator ? '' : 'heading-showCreator'}
```
Comment (silwalanish) [inline src/Components/ReconstructionCard.jsx:30]: Name class appropriately.
```suggestion
                className={!this.props.showCreator ? 'heading-creator-hidden' : ''}
```

**Review by silwalanish — APPROVED** [review]

**Review by silwalanish — APPROVED** [review]

**Pull request merged**